Assets/Nickel/NickelBalanace/QuickOutline/Scripts/Outline.cs

```diff
@@ -0,0 +1,277 @@
+﻿//
+//  Outline.cs
+//  QuickOutline
+//
+//  Created by Chris Nolet on 3/30/18.
+//  Copyright © 2018 Chris Nolet. All rights reserved.
```

Comment:
Everything related to this code that you got is not namespaced correctly. Fix it.